code/core/src/common/utils/cli.ts

```diff
@@ -156,4 +157,5 @@ export const createLogStream = async (
   });
 };
 
-export const isCorePackage = (pkg: string) => Object.keys(versions).includes(pkg);
+export const isCorePackage = (pkg: string) => Object.keys(storybookPackagesVersions).includes(pkg);
```

Comment:
```suggestion
export const isCorePackage = (pkg: string) => !!(storybookPackagesVersions[pkg]);
```

This should be slightly faster, and maybe simpler?